packages/client-core/src/admin/common/Table.tsx

```diff
@@ -112,14 +115,16 @@ const DataTable = ({ query, columns, rows }: DataTableProps) => {
       ))}
     </div>
   ) : (
-    <div className="relative h-full">
+    <div className="relative">
       {query.status === 'pending' && (
         <div className="absolute left-1/2 top-1/2 flex h-8 -translate-x-1/2 -translate-y-1/2 items-center">
           <LoadingView className="mx-1 h-8 w-8" />
           <Text className="mx-1">{t('common:table.refetching')}</Text>
         </div>
       )}
-      <Table containerClassName={`${query.status === 'pending' && 'opacity-50'} h-[calc(100%_-_160px)]`}>
+      <Table
+        containerClassName={twMerge(`${query.status === 'pending' && 'opacity-50'} table-size-${size}`, className)}
```

Comment:
since we are using `table-size-` tailwind utility at only 1 place, we can convert the size chart to a record object
for example,
```
{
 lg: 'h-[calc(100vh - 386px)]'
...
}
```

Reply:
Thanks, the solution is above